WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+import Gridicons
+
+/// A view that consists of the frame of a Dashboard card
+/// Title, icon and action can be customizable
+class BlogDashboardCardFrameView: UIView {
+
+    /// The main stack view, in which the header and the content
+    /// are appended to.
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView()
```

Comment:
What do you think of adding the arranged subviews here instead of down below in init?
```suggestion
        let stackView = UIStackView(arrangedSubviews: [headerStackView])
```

Reply:
Hmmm, I'm not sure if I like that because the whole view initialization is very implied and hard to read. I prefer to keep it only layout-related.

For example, by reading in the future to understand the layout you would have to read `mainStackView` to see that `headerStackView` is added, and then read `headerStackView` to understand which views are added. While on the init is more clear.

I changed that a bit in e97b68b6c0253fdf33cfb99a464721520f602c69 moving to it's own method, let me know what do you think!